.github/workflows/linux.yaml

```diff
@@ -57,7 +57,7 @@ jobs:
       nightly: ${{ github.event.inputs.nightly }}
       modular: true
   linux-coverage:
-    if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
+    if: github.ref == 'refs/heads/main' && (github.event_name == 'push' || github.event_name == 'pull_request')
```

Comment:
Any reason we don't want to run coverage on the feature/* branches?  Also, I think we want to run this on workflow_dispatch event.

Reply:
I don't think so. If you want to see coverage you can either run it locally or create a PR. Two very valid workarounds.

Reply:
> Also, I think we want to run this on workflow_dispatch event.

Done